packages/kbn-grid-layout/grid/grid_row.tsx

```diff
@@ -40,7 +41,9 @@ export const GridRow = forwardRef<
     gridRef
   ) => {
     const currentRow = gridLayoutStateManager.gridLayout$.value[rowIndex];
-    const [panelIds, setPanelIds] = useState<string[]>(Object.keys(currentRow.panels));
+    const [panelIds, setPanelIds] = useState<string[]>(
+      Object.keys(getKeysInOrder(currentRow.panels))
+    );
```

Comment:
Looks like this is returning the indexes of the array and not the panel IDs, so we're getting the following error:

![image](https://github.com/user-attachments/assets/bd171835-9037-4f5a-9f9f-0ffc6185b5c2)

```
grid_panel.tsx:41 Uncaught TypeError: Cannot read properties of undefined (reading 'column')
    at grid_panel.tsx:41:1
    at mountMemo (react-dom.development.js:15846:1)
    at Object.useMemo (react-dom.development.js:16219:1)
    at useMemo (react.development.js:1532:1)
    at grid_panel.tsx:39:1
```